source/Calamari.CloudAccounts/Calamari.CloudAccounts.nuspec

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why have we added nuspec files for these?

Reply:
I suspect i may have something to do with

Reply:
Previously we had been using `DotNetCorePack` but that only allows you to specify a project, and not a published set of files. `NuGetPack` is the method that is used for the other packages that were already successfully signing the binaries so I thought for consistency I'd use the same here. And the `NuGetPack` method requires a nuspec file. 

The process seems to be publish -> sign -> pack, and `NuGetPack `was the only way I found that supported that - it could be that I'm missing something though, I found the Cake documentation to be pretty sparse and hard to navigate.

Reply:
> Perhaps elaborating a bit more in the PR description might help clarify the changes made?

Missed this comment first time around. I've now updated the PR description to include the same details as in my comment above.